modules/auxiliary/gather/android_stock_browser_iframe_dos.rb

```diff
@@ -0,0 +1,64 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Android Stock Browser Iframe DOS",
+      'Description'    => %q{
+        This module exploits a vulnerability in the native browser that comes with Android 4.0.3.
+        If successful, the browser will crash after viewing the webpage.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         => [
+        'Jean Pascal Pereira',  # Original exploit discovery
+        'Jonathan Waggoner'     # Metasploit module
+      ],
+      'References' => [
+        [ 'PACKETSTORM', '118539'],
+        [ 'CVE', '2012-6301' ]
+      ],
+      'Platform'            => 'android',
+      'Arch'                => ARCH_DALVIK,
+      'DefaultOptions'      => { 'PAYLOAD' => 'android/shell/reverse_http' },
+      'Targets'             => [ [ 'Automatic', {} ] ],
+      'DisclosureDate'      => "Dec 1 2012",
+      'DefaultTarget'       => 0
```

Comment:
I believe you can and should remove `Platform`, `Arch`, `DefaultOptions`, `Targets` and `DefaultTarget`, none of which are used by auxiliary modules.